docs/fundamentals/connection/connection-options.txt

```diff
@@ -0,0 +1,343 @@
+               'appName' => 'myLaravelApp',
+               'compressors' => 'zlib',
+               'zlibCompressionLevel' => 7,
+           ],
+       ],
+   ],
+
+To specify options as parameters in the connection string, use the following
+query string syntax formatting:
+
+- Add the question mark character, ``?``, to separate the host information
+  from the parameters.
+- Add the options by formatting them as ``<option>=<value>``.
+- Insert the ampersand character, ``&``, between each option and value pair
+  to separate them.
+
+The following setting example shows the connection string parameter syntax:
+
+.. code-block:: php
+
+   'dsn' => 'mongodb+srv://mongodb0.example.com/?appName=myLaravelApp&compressors=zlib',
+
+Option Descriptions
+~~~~~~~~~~~~~~~~~~~
+
+The following table describes a list of connection and authentication options
+and their default values:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 12 12 46
+
+   * - Option Name
+     - Accepted Values
+     - Default Value
+     - Description
+
+   * - **appName**
+     - String
+     - None
+     - | Specifies the application name that the {+php-library+} sends the
+         MongoDB deployment as part of the handshake.
+       | Specifying ``appName`` can help you identify activities related
+         to that application in the server logs.
+
+   * - **authMechanism**
+     - String
+     - None
+     - Specifies which authentication mechanism to use. If you do not
+       specify this option, the driver uses the default authentication
+       mechanism. To learn more, see :manual:`Authentication </core/authentication/>`
+       in the {+server-docs-name+}.
+
+   * - **authMechanismProperties**
+     - String
+     - None
+     - Specifies more properties related to the authentication mechanism set in
+       the ``authMechanism`` option.
+
+   * - **authSource**
+     - String
+     - See description
+     - | Specifies the database used to authenticate.
+       | This option defaults to ``admin`` for SCRAM-based authentication
+         mechanisms, ``$external`` for the ``MONGODB-X509`` mechanism, and the
+         database name or ``$external`` for the ``PLAIN`` mechanism.
+
+   * - **compressors**
+     - A comma-separated list of strings
+     - None
+     - Specifies data compressors that the {+php-library+} uses to reduce the
+       amount of network data passed between MongoDB and your application in
+       the specified order.
+
+   * - **connectTimeoutMS**
+     - Non-negative integer
+     - ``10000`` (10 seconds)
+     - Specifies the connection timeout, in milliseconds, passed to each
+       underlying TCP stream when attempting to connect to the server.
+
+   * - **directConnection**
+     - Boolean
+     - ``false``
+     - Specifies whether to directly connect to a single host instead of
+       discovering and connecting to all servers in the cluster.
+
+   * - **heartbeatFrequencyMS**
+     - Integer greater than or equal to 500
```

Comment:
S: since you use monospace for most of the integer values in the table, I think this should be monospaced too
```suggestion
     - Integer greater than or equal to ``500``
```

Reply:
Applies elsewhere in the table (the localThresholdMS, maxIdleTimeMS, and zlibCompressionLevel rows)

Reply:
Good catch, thanks!